pkgs/test/README.md

```diff
@@ -233,6 +236,19 @@ dart test --test-randomize-ordering-seed=random
 Setting `--test-randomize-ordering-seed=0` will have the same effect as not
 specifying it at all, meaning the test order will remain as-is.
 
+### Selecting a Test Reporter
+
+You can adjust the output format of the test results using the `--reporter=<option>`
+command line flag. The default format is the `compact` output format - a single
+line, continuously updated as tests are run. When running on the GitHub Actions CI
+however, the default changes to the `github` output format - a reporter customized
```

Comment:
Pondering if we should be explicit abouth the ENV var we check. So it's clear it's not magic.